### PR TITLE
Reverse sidecar 

### LINKED ISSUE #1689: Reverse the Sidecar Injection Feature
**Description**

When Istio is present, we should clearly mention to our user the status of the Istio Sidecar injection. 
By default we should have it enabled. Both states will add the `istio-injection: disabled` or `istio-injection: enabled` label to the namespace
The toggle shows up in simple mode too.

**Reasons**

**Stretch Goal**

- [ ] Have a toggle to disable / enable sidecar injection at Deployment level
- [ ] Invert the Checkbox at JOB level too

**Attachments**

image

### PR DESCRIPTION
**Pull request by wchrapka**
**Description**

Changes proposed in this pull request:

- always include the sidecar injection label enabled/disabled
- remove istio switch from jobs
- add istio switch to deployments

**Related issue(s)**
Resolves #1689

### CODE REVIEW

**Review by qbalukom — CHANGES_REQUESTED** [review]

File: core-ui/src/resources/Namespaces/NamespaceCreate.js [hunk 1]
```diff
@@ -13,12 +13,14 @@ import { createResourceQuotaTemplate } from 'resources/ResourceQuotas/templates'
 import { MemoryInput } from './MemoryQuotas';
 import { createNamespaceTemplate } from './templates';
 import { LimitPresets, MemoryPresets } from './Presets';
+import { useSidecar } from 'shared/hooks/useSidecarInjection';
 import { CONFIG } from './config';
 
 import './NamespaceCreate.scss';
 
 const ISTIO_INJECTION_LABEL = 'istio-injection';
```
[the remark this answers is not in the document]
Comment (wchrapka) [reply core-ui/src/resources/Namespaces/NamespaceCreate.js:21]: This is a standard practice for defining constants that are dependent on some external sources. We use it `run-prepare-functions.spec`

File: core-ui/src/resources/Deployments/DeploymentCreate.js [hunk 2]
```diff
@@ -69,6 +83,18 @@ export function DeploymentCreate({
       initialResource={initialDeployment}
       handleNameChange={handleNameChange}
     >
+      <ResourceForm.FormField
+        advanced={!isIstioFeatureOn}
```
[the remark this answers is not in the document]
Comment (wchrapka) [reply core-ui/src/resources/Deployments/DeploymentCreate.js]: Checked and changed - the component is hidden when Istio is off

File: core-ui/src/resources/Deployments/DeploymentCreate.js [hunk 3]
```diff
@@ -69,6 +83,20 @@ export function DeploymentCreate({
       initialResource={initialDeployment}
       handleNameChange={handleNameChange}
     >
+      {isIstioFeatureOn ? (
+        <ResourceForm.FormField
+          simple
```
Comment (qbalukom) [inline core-ui/src/resources/Deployments/DeploymentCreate.js:88]: is it supposed to be simple-only then?
i believe this flag should be removed completely

**Review by qbalukom — CHANGES_REQUESTED** [review]

**Review by qbalukom — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**